cylc/uiserver/authorise.py

```diff
@@ -165,12 +165,12 @@ def __init__(
         log,
     ):
         self.owner_user_name: str = owner_user_name
-        self.owner_user_groups: List[str] = self._get_groups(
-            self.owner_user_name
-        )
         self.log = log
         self.owner_auth_conf: dict = owner_auth_conf
         self.site_auth_config: dict = site_auth_conf
+        self.owner_user_groups: List[str] = self._get_groups(
+            self.owner_user_name
+        )
```

Comment:
I don't know how to reproduce it, but the cause should be fairly self-evident from looking at `self._get_groups()`:
https://github.com/cylc/cylc-uiserver/blob/ef4a4d1ec613dec6aba6e6446f530fbf285659d4/cylc/uiserver/authorise.py#L487-L492

Can't use `self.log` before it's been defined